src/vm/eventpipebuffermanager.cpp

```diff
@@ -1044,6 +1044,24 @@ void EventPipeBufferManager::SuspendWriteEvent(uint32_t sessionIndex)
     }
 }
 
+void EventPipeBufferManager::ResumeWriteEvent()
```

Comment:
If we move Suspend to occur later we won't need to resurrect the code for resume.